cmd/whclient/main.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"io"
+	"net"
+	"os"
+	"strconv"
+	"sync"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/taskcluster/taskcluster-client-go"
+	"github.com/taskcluster/taskcluster-client-go/auth"
+	"github.com/taskcluster/webhooktunnel/util"
+	"github.com/taskcluster/webhooktunnel/whclient"
+)
+
+// whclient <clientID> <accessToken> <targetPort>
+const usage = `whcli <clientID> <accessToken> <targetPort>`
```

Comment:
Consider using https://github.com/docopt/docopt.go

There is a tool here for constructing the docopt string:
http://try.docopt.org/   also docs: http://docopt.org/

It's pretty nice that way as we get to have some usage documentation and ideally some paragraphs explaining what this utility does...